src/functions/routers/types.ts

```diff
@@ -42,12 +41,12 @@ type FunctionData<InputParameters extends FunctionParameters> = {
   inputs: InputParameters;
 };
 
-export type BlockAction = {
-  type: string;
-  block_id: string;
-  action_id: string;
-  // deno-lint-ignore no-explicit-any
-  [key: string]: any;
+type Interactivity = {
+  interactor: {
+    secret: string;
+    id: string;
+  };
```

Comment:
nit: Should we have a user context type that we set interactor to, or do you think that's not valuable since we're (currently) not surfacing user context separately?

I'm currently thinking that we'll probably want a type for user context at some point and think that this should be tied to it whenever we do.

Reply:
Possibly, I'll go ahead and break it out for now, even for clarity it might be nice. I'm not sure I see an immediate payload that would include just that piece, but still a good idea.